charts/cortex/templates/store-gateway/store-gateway-svc.yaml

```diff
@@ -16,5 +17,39 @@ spec:
       protocol: TCP
       name: http-metrics
       targetPort: http-metrics
+    - port: {{ .Values.config.server.grpc_listen_port }}
```

Comment:
Same comments as for the ingester. I do not believe we should mix the services for load balancing with the headless service used by the statefulset. Thus I also do believe that we should keep the grpc as part of the headless service. I believe the grpc port is used for discovery if memberlist is configured instead of Consul.